src/frontend/Sider/Behandling/Simulering/simuleringUtils.ts

```diff
@@ -10,3 +10,13 @@ export const harVedtaksresultatMedTilkjentYtelse = (
             return false;
     }
 };
+
+export const finnDefaultÅrForSimuleringsvisning = (muligeÅr: number[]) => {
+    const inneværendeÅr = new Date().getFullYear();
+    if (muligeÅr.includes(inneværendeÅr)) {
+        return inneværendeÅr;
+    } else if (Math.max(...muligeÅr) < inneværendeÅr) {
+        return Math.max(...muligeÅr);
```

Comment:
Må man håndtere at muligeÅr er en tom liste? :P 
```suggestion
Math.max(...[]) // -Infinity
```

Reply:
Og 
```js
Math.min(...[]) // Infinity
```

Reply:
Kom på at vi sikkert ikke trenger dette, fordi hvis listen er tom, så tror jeg ikke vi viser noen simulering. Det er noe tekst i stedet. 
Så du kan ignorere de tidligere kommentarene